tools/inspector/elements/record-button.js

```diff
@@ -69,7 +69,7 @@ export class RecordButton extends LitElement {
     const blob = new Blob([entries.join('\n')], {type: 'octet/stream'});
     const url = window.URL.createObjectURL(blob);
     a.href = url;
-    a.download = 'mozillavpn.txt';
```

Comment:
Because we already have logs, I suggest removing the .txt file.

Reply:
So this comment has not been addressed. Now the user will have 2 files. This is bad. @mcleinman follow up?